test/system/src/main/java/io/pravega/test/system/framework/services/docker/DockerBasedService.java

```diff
@@ -97,21 +119,22 @@ public boolean isRunning() {
             if (!serviceList.isEmpty()) {
                 long replicas = Exceptions.handleInterruptedCall(
                         () -> dockerClient.inspectService(serviceList.get(0).id()).spec().mode().replicated().replicas());
-                log.info("Replicas {}", replicas);
-                log.info("Task running count {}", taskRunningCount);
+                log.info("isSynced:Replicas {}", replicas);
+                log.info("isSynced:Task running count {}", taskRunningCount);
                 if (((long) taskRunningCount) == replicas) {
                     return true;
                 }
             }
         } catch (DockerException e) {
             log.error("Unable to list docker services", e);
         }
+        log.info ("returning false from isSynced:");
         return value;
```

Comment:
If `value = false` and it is not changed, you can just remove the `value` parameter and return `false` here directly.